### PR TITLE
Add a beginning list of example grants

### PR DESCRIPTION
**Pull request by ericholscher**
Hopefully we can build on this over time, but I wanted to at least get it started :)

### CODE REVIEW

File: resources.md [hunk 1]
```diff
@@ -14,5 +14,12 @@ Writing a grant proposal can be a daunting task for first-time writers. Here is
 - [Nonprofit Grant Writing: How to Secure Grants for Your Cause](https://grantsplus.com/nonprofit-grant-writing/)
 - [Developing a Logic Model or Theory of Change](https://meta.wikimedia.org/wiki/Learning_and_Evaluation/Logic_models#Steps_to_creating_a_logic_model)
 
+## Example grants
```
Comment (pradyunsg) [inline resources.md]:
```suggestion
## Example grant applications
```

Maybe?

Comment (ericholscher) [reply resources.md]: 🙏 Thanks :)

File: resources.md [hunk 2]
```diff
@@ -14,5 +14,12 @@ Writing a grant proposal can be a daunting task for first-time writers. Here is
 - [Nonprofit Grant Writing: How to Secure Grants for Your Cause](https://grantsplus.com/nonprofit-grant-writing/)
 - [Developing a Logic Model or Theory of Change](https://meta.wikimedia.org/wiki/Learning_and_Evaluation/Logic_models#Steps_to_creating_a_logic_model)
 
+## Example grant applications
+
+We are working to collect submitted grants from the community, so that you can get a better idea of what is required.
+This section will contain grant proposals for programs that we list in our [funders list](funders.md].
```
Comment (xmunoz) [inline resources.md]:
```suggestion
This section will contain grant proposals for programs that we list in our [funders list](funders.md).
```

**Review by xmunoz — APPROVED** [review]

**Comment by ericholscher** [comment]
@xmunoz Thanks for reviewing this so quickly 🎉

**Pull request merged**